src/main/java/nextstep/ladder/domain/Lines.java

```diff
@@ -0,0 +1,73 @@
+package nextstep.ladder.domain;
+
+import java.util.List;
+import java.util.Objects;
+
+public class Lines {
+
+    private List<Line> lines;
+
+    public Lines(List<Line> lines, int maxNum, LineCreationStrategy lineCreationStrategy) {
+        this.lines = lines;
+        this.lines = drawLine(maxNum, lineCreationStrategy);
+    }
```

Comment:
```suggestion
    private final List<Line> lines;

    private Lines(List<Line> lines) {
        this.lines = lines;
    }
```
이전 리뷰에도 남겼는데 이렇게 표현할 수 있도록 개선해보세요!
지금은 코드가 무척 부자연스러워보이네요